src/source/empty.rs

```diff
@@ -1,12 +1,17 @@
-use crate::Sample;
-use crate::Source;
-use std::marker::PhantomData;
-use std::time::Duration;
+use crate::{Sample, Source};
+use std::{marker::PhantomData, time::Duration};
```

Comment:
I don't like this change, can you keep it? The sample, Source change is fine, only really about the second line here.